pages/Configuring/Master-Layout.md

```diff
@@ -4,7 +4,7 @@ if you want to use anything other than the default left/right split.
 
 # Quirks
 
-The right, "slave" windows will always be split uniformly. You cannot change
+The right, "slave" windows will be split uniformly. You can change
```

Comment:
The right, "slave" windows will be split uniformly by default, but you can change their size manually though.